test/Chainweb/Test/Pact/Utils.hs

```diff
@@ -201,6 +245,10 @@ sender00KeyPair =
     , ED25519
     )
 
+sender00 :: SimpleKeyPair
```

Comment:
For instance, the txg had some infrastructure for reading these keys directly from the JSON files.

https://github.com/kadena-io/txg/blob/38c1112ff74c07d41e5924fb252b8eda28eda2b0/lib/TXG/Simulate/Contracts/Common.hs#L120

Reply:
That I've deliberately avoided as it means keyset manipulation is in `IO` ...